### ISSUE TITLE
Can't resolve 'preact-context'

### ISSUE THREAD

**Issue by RodionNikolaev** [developit/preact-redux#46]
"preact-redux": "2.1.0" (2.0.3 works fine),

ERROR in ./node_modules/preact-redux/dist/preact-redux.esm.js
Module not found: Error: Can't resolve 'preact-context' in .\node_modules\preact-redux\dist'
 @ ./node_modules/preact-redux/dist/preact-redux.esm.js 2:0-47 55:17-30

**Comment by fjorgemota** [comment]
`preact-context` is a new peer dependency that you should add to your own project, so you can, for example, reuse it while using Preact 8 (avoiding duplicates if you already use preact-context on your project, for example).

It was added as a peer dependency to this project because Redux version was updated and it needs React's Context API to work, which the `preact-context` dependency implements successfully to Preact users.

Basically, to fix that problem you should run, on your project, `npm install --save preact-context`. Understood?

**Comment by fjorgemota** [comment]
I consider this issue closed after the answer I posted yesterday.

If you have any new problems with the library, please, open a new issue. Or, if this same problem persists, please reopen the issue.

Thanks. :)
